### PR TITLE
using new matrix aliases, removing ublas.

### PR DESCRIPTION
**Pull request by jcotela**
Changes from #1931 in the ThermoMechanicalApplication

### CODE REVIEW

File: applications/ThermoMechanicalApplication/custom_elements/SUPG_conv_3d.cpp [hunk 1]
```diff
@@ -1,22 +1,22 @@
 // Kratos Multi-Physics
```
Comment (RiccardoRossi) [inline applications/ThermoMechanicalApplication/custom_elements/SUPG_conv_3d.cpp]: you may want to use the new header here

**Review by RiccardoRossi — DISMISSED** [review]

**Comment by jcotela** [comment]
@RiccardoRossi I've changed the headers for all the elements. I tried to give proper attribution based on file history, apologies in advance if I have missed someone.

File: applications/ThermoMechanicalApplication/custom_elements/Poisson_3d.h [hunk 2]
```diff
@@ -93,7 +55,7 @@ namespace Kratos
 /**
 ASGS is an abriviation for Algebraic Sub-Grid Scale element. It is implemented to solve
 Implicitly the NS equations in a variotionally consistant sub-grid scale methid. It also has the OSS swith
-to use Orthogonal Sub Scales to use impose explicity the orthogonality condition on subscales� estimation.
+to use Orthogonal Sub Scales to use impose explicity the orthogonality condition on subscales� estimation.
```
Comment (RiccardoRossi) [inline applications/ThermoMechanicalApplication/custom_elements/Poisson_3d.h]: strange symbol

**Review by RiccardoRossi — DISMISSED** [review]

**Comment by jcotela** [comment]
@RiccardoRossi removed weird characters. The strange thing is that I thought we had already removed those on a separate PR...

**Review by RiccardoRossi — APPROVED** [review]

**Pull request merged**